README.md

```diff
@@ -132,12 +132,43 @@ currently, there are verification tasks for the ILP32 (32-bit) and the LP64 (64-
 
 In order to obtain verification tasks from the programs and specifications in the repository,
 a simple task-definition mechanism is used.
-For each program, the repository contains a .yml file that specifies 
-  - the subject files (the parts that a program consists of),
-  - the results (for each specification of the program, the expected outcome is indicated), and
-  - the parameters to be used for verification.
+For each program, the repository contains a .yml file that specifies the following items:
+  - `format_version`: the version of the format (a version string)
+  - `input_files`: the subject program files or directories
+    (a file or directory name, or a list of files or directory names, that the program consists of)
+  - `properties`: the properties that constitute the specification of the program,
+    each consisting of the following items:
+    - `property_file`: file that contains a property definition
+    - `expected_verdict`: the intended verification result (`true` or `false`)
+    - `subproperty` (optional): a subproperty of the property that is violated
+      in cases where the property is a conjunction of subproperties (for verdict `false`)
+  - `options`: parameters that are relevant for verification or give extra information:
+    - `language`: programming language that the program is written in (`C` or `Java`)
+    - `data_model` data model of the computer architecture
+      (`ILP32`, `LP64`, see http://www.unix.org/whitepapers/64bit.html)
```

Comment:
Is this list meant to be exhaustive, i.e., we do not allow other values for the `data_model`? It would make sense to allow to add tasks that have a different data model, but only use those with either `ILP32` or `LP64` for the competition?

Will I be able to filter the data_model or language in a benchmark definition XML? @PhilippWendler this could be generically supported by BenchExec I guess, or is it even already possible?

Reply:
There is no such filter feature in BenchExec, and adding it is a little bit tricky because we would need to define a language for filter expressions.

If this is desired, I would as a first iteration suggest to do filtering outside of BenchExec in a script that uses `yq` and dumps appropriate `.set` files. Teen later on we can maybe add such a feature to BenchExec.

Reply:
In principle, other values for data_model are allowed, but not for SV-COMP and Test-Comp.